shopify_tool/templates/barcode_label/style.css

```
@@ -0,0 +1,80 @@
+  border-top: 0.3mm solid black;
+  padding-top: 0.5mm;
+  margin-top: 0.5mm;
+  overflow: hidden;
+}
+
+.tag-row .field-label { width: 9mm; font-size: 3mm; font-weight: bold; }
+.tag-row .field-value {
+  font-weight: bold;
+  overflow: hidden;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)